gubernator/testgrid.py

```diff
@@ -104,14 +105,17 @@ def path_to_query(path):
     options = {}
     for dashboard in get_config().get('dashboards', []):
         dashboard_name = dashboard['name'][0]
-        for tab in dashboard['dashboard_tab']:
+        tabs = dashboard['dashboard_tab']
+        for tab in tabs:
+            if 'base_options' in tab:
```

Comment:
What sort of options do you expect all tabs to have? Do you expect all tabs to have regex filters?

It's easy enough to change in the future when this changes.